README.md

```diff
@@ -3,32 +3,47 @@
 
 ###### [API](https://github.com/microsoft/playwright/blob/v0.10.0/docs/api.md) | [FAQ](#faq) | [Contributing](#contributing)
 
-
 Playwright is a Node library to automate the [Chromium](https://www.chromium.org/Home), [WebKit](https://webkit.org/) and [Firefox](https://www.mozilla.org/en-US/firefox/new/) browsers with a single API. It enables **cross-browser** web automation that is **ever-green**, **capable**, **reliable** and **fast**.
 
-|          | ver | Linux | macOS | Win |
+|          | Version | Linux | macOS | Windows |
 |   ---:   | :---: | :---: | :---:  | :---: |
 | Chromium| 81.0.4044 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 | WebKit | 13.0.4 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| Firefox |73.0b3 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-- Headless is supported for all the browsers on all platforms.
+| Firefox | 73.0b3 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+Headless is supported for all the browsers on all platforms.

```

Comment:
What about Internet Explorer?

Reply:
at this point, Playwright does not support any other browsers. we can always update the readme if/when that changes